lib/adapter.js

```diff
@@ -375,7 +375,16 @@ module.exports = (function() {
 
         // Run Query
         client.query(query.query, query.values, function __CREATE__(err, result) {
-          if(err) return cb(err);
+          if (err) {
+            return cb(err);
+          }
+
+          // Use this in tests to detect whether a test inserted rows into the
+          // database. After you clear the database, reset DBRecordsInserted to
+          // false.
+          if (process.env.NODE_ENV === 'test' && result.rows.length > 0) {
+            global.DBRecordsInserted = true;
```

Comment:
Hmm. It seems more cumbersome to hack a global var into the module. How many prototypes would you have to spy on?